spark/src/main/java/com/netflix/iceberg/spark/source/IcebergSource.java

```diff
@@ -59,7 +60,13 @@ public String shortName() {
   public DataSourceReader createReader(DataSourceOptions options) {
     Configuration conf = new Configuration(lazyBaseConf());
     Table table = getTableAndResolveHadoopConfiguration(options, conf);
-    return new Reader(table);
+
+    // get caseSensitive from Conf or default
+    boolean caseSensitive = ConfigProperties.isCaseSensitive(conf);
```

Comment:
Instead of having an Iceberg property, I think this should check spark.sql.caseSensitive in the current session's SQLConf:

```java
String caseSensitive = lazySparkSession().conf().get("spark.sql.caseSensitive")
```

Reply:
Ah, much simpler. Will do.